src/Resources/config/services.yml

```diff
@@ -1,5 +1,6 @@
 parameters:
   env(TURBOLINKS_ENABLED): false
+  recaptcha.enabled: "%env(bool:RECAPTCHA_ENABLED)%"

```

Comment:
Unlike the Turbolinks flag, this parameter deliberately does not have a default, so the env variable will be required. My aim was to prevent a site being pushed live with Recaptcha accidentally disabled.